src/layers/base.jl

```diff
@@ -174,6 +174,8 @@ function gelu_forward_backward(x)
     return (forward, backward)
 end
 
+gelu_erf_forward_backward(x) = (gelu_erf(x), NNlib.deriv_gelu_erf(x))
```

Comment:
```suggestion
using SpecialFunctions: erf

function gelu_erf_forward_backward(x)
    TWO = oftf(x,2)
    SQRT2 = sqrt(TWO)
    X_RSQRT2 = x / SQRT2
    ERF_SQRT1 = 1 + erf(X_RSQRT2)
￼   Φ = ERF_SQRT1 / TWO
    HALF_X = x / TWO
￼   forward = HALF_X * ERF_SQRT1
    backward = Φ + X_RSQRT2 * exp(-x * HALF_X) / sqrt(oftf(x,π))
    return (forward, backward)
end
```
And add a dependency for SpecialFunctions.jl